modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -479,4 +518,114 @@ export default class NestedTabs extends Base {
 
 		return !! $tabTitleContainerElement && isTabTitleActive ? true : false;
 	}
+
+	changeScrollStatus( event ) {
+		const slider = this.elements.$headingContainer[ 0 ];
+
+		if ( 'mousedown' === event.type ) {
+			slider.classList.add( 'e-scroll' );
+			slider.dataset.pageX = event.pageX;
+		} else {
+			slider.classList.remove( 'e-scroll' );
+			slider.dataset.pageX = '';
+		}
+	}
+
+	isHorizontalScroll() {
+		const slider = this.elements.$headingContainer[ 0 ];
+
+		return slider.clientWidth < this.getChildrenWidth( slider.children ) && 'enable' === this.getHorizontalScrollSetting();
+	}
+
+	getChildrenWidth( children ) {
+		let totalWidth = 0;
+
+		const parentContainer = children[ 0 ].parentNode,
+			computedStyles = getComputedStyle( parentContainer ),
+			gap = parseFloat( computedStyles.gap ) || 0; // Get the gap value or default to 0 if it's not specified
+
+		for ( let i = 0; i < children.length; i++ ) {
+			totalWidth += children[ i ].offsetWidth + gap;
+		}
+
+		return totalWidth;
+	}
+
+	setHorizontalScrollAlignment( event = {} ) {
+		if ( ! this.elements ) {
+			return;
+		}
+
+		const slider = this.elements.$headingContainer[ 0 ],
+			tabsDirection = this.getTabsDirection();
```

Comment:
`tabsDirection` can be inside the if statement to avoid an unnecessary call to this function? You only need `slider` in both cases.